// react-lorem-ipsum/dist/index.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
Object.defineProperty(exports, "Avatar", {
  enumerable: true,
  get: function get() {
    return _user.Avatar;
  }
});
Object.defineProperty(exports, "LoremIpsum", {
  enumerable: true,
  get: function get() {
    return _loremIpsum.LoremIpsum;
  }
});
exports["default"] = void 0;
Object.defineProperty(exports, "fullname", {
  enumerable: true,
  get: function get() {
    return _user.fullname;
  }
});
Object.defineProperty(exports, "loremIpsum", {
  enumerable: true,
  get: function get() {
    return _loremIpsum.loremIpsum;
  }
});
Object.defineProperty(exports, "name", {
  enumerable: true,
  get: function get() {
    return _user.name;
  }
});
Object.defineProperty(exports, "surname", {
  enumerable: true,
  get: function get() {
    return _user.surname;
  }
});
Object.defineProperty(exports, "username", {
  enumerable: true,
  get: function get() {
    return _user.username;
  }
});

var _loremIpsum = require("./lorem-ipsum");

var _user = require("./user");

var _default = _loremIpsum.LoremIpsum;
exports["default"] = _default;
